addons/core/functions/fnc_ClientInit.sqf

```diff
@@ -232,9 +224,6 @@ player addEventHandler ["killed", {
     call TFAR_fnc_hideHint;
 }];
 
-call TFAR_fnc_processRespawn; //Handle our current spawn
```

Comment:
Might break speak volume HUD indicator. Needs testing.

Reply:
If you provide details how volume HUD indicator sholud work I'll test it.

Reply:
It's a HUD indicator that shows the voice volume. You can enable it in CBA settings